mod/metric_answer_lookup/set/type_plus_right/metric_value/value.rb

```diff
@@ -1,3 +1,5 @@
+card_accessor :unknown
```

Comment:
It's a field of value because I consider it as an interface to change the value. It's not data that we use. Actually, unknown fields are not stored in the database.  And our api doesn't support to manipulate siblings very well. It's more straight forward that +value checks if there is a +unknown subfield than to check if a +unknown is part of the same act.